### DISCUSSION TITLE
Bezier curves

### DISCUSSION THREAD

**Discussion by probablycorey** [replit/kaboom#393] [Q&A]
Does anyone have any tips or suggestions for making a bezier curve with Kaboom? Something similar to https://p5js.org/reference/#/p5/bezier?

**Comment by slmjkdbtl** [comment]
It's planned to support curves built-in, but you can implement yourself with `drawLine()`

There is an example but the code is kinda messy atm, and it's using game objects instead of doing as a rendering function: https://replit.com/@slmjkdbtl/markier-curve?v=1
